Dockerfile

```diff
@@ -6,14 +6,13 @@ RUN curl -o- -L https://yarnpkg.com/install.sh | bash
 # Copy package dependencies
 WORKDIR /home/node/html/
 COPY package.json package.json
-COPY yarn.lock yarn.lock
```

Comment:
It could still be helpful to include the `yarn.lock` when it's there, otherwise dependencies will be floating, which can cause its own problems.